intelmq/bots/BOTS

```diff
@@ -107,7 +107,10 @@
                 "name": "__FEED__",
                 "provider": "__PROVIDER__",
                 "rate_limit": 3600,
-                "ssl_client_certificate": null
+                "ssl_client_certificate": null,
+                "verify_gpg_signatures": false,
```

Comment:
This might be nitpicking, but the format is called PGP, not GPG (which is an implementation). Also applies to `gpg_signature_suffix`